demos/twa-webview-fallback/src/main/res/layout/activity_web_view_fallback.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".WebViewFallbackActivity">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
nit: lack of newline.

Reply:
Also if this class is to remain empty, you can just use `/>` on the end of line 7 and get rid of line 9.

Reply:
Removed the entire layout, as I'm creating the WebView programatically.